Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -152,4 +153,35 @@ + (NSNumber *)getScreenHeight {
     return [NSNumber numberWithInteger:(NSInteger)height];
 }
 
++ (NSDictionary *)getListOfApps {
+    NSMutableArray *appsPresent = [[NSMutableArray alloc] init];
+    NSMutableArray *appsNotPresent = [[NSMutableArray alloc] init];
+    NSDictionary *appsData = [NSDictionary dictionaryWithObjects:@[appsPresent, appsNotPresent] forKeys:@[@"canOpen", @"notOpen"]];
```

Comment:
Hah, yah, i am aware of this one. Just from habbit and used to XCode's intellisense :-)